test/integration/get-capability-handles-transient.int.c

```diff
@@ -189,9 +189,10 @@ test_invoke (TSS2_SYS_CONTEXT *sapi_context)
     TPM2_HANDLE          *handles_load;
     TPM2_HANDLE          *handles_query;
     size_t               handles_count;
-    char                *env_str         = NULL, *end_ptr = NULL;
+    char                 *env_str        = NULL, *end_ptr = NULL;
```

Comment:
Doesn't look like any functional change on this line. Just whitespace noise?

Reply:
The column is not aligned with any of the other lines.  I will restore the error.

Reply:
I've learned the hard way that spacing things like this is a terrible idea (but "it seemed like a good idea at the time"). Well-intentioned folks like yourself try to align things or clean up alignment that got out of wack and just end up creating messy patches. If you want to clean this stuff up then just remove it and please do so in a patch dedicated to the cleanup.

Reply:
I think it's silly to leave formatting problems unfixed. It makes the code harder to read.
Some code is indented 4 spaces, some 2, or 3.  Some functions have a mixture of indentation.
I see other pull requests here by everyone else that fix formatting.

But I restored it to the way it originally was--so the issue is mute.